gpfit/__init__.py

```diff
@@ -1 +1,4 @@
 "Package for GP-compatible data fitting"
+__version__ = "1.0.0"
```

Comment:
@pgkirsch mentioned he might want to change this back to 0.2? No preference from me, just flagging it to double check

Reply:
Thanks for flagging with go through and make sure it's 0.2 everywhere.